src/main.js

```diff
@@ -16,7 +16,7 @@ const { onConnect, onFirefoxConnect } = require("./utils/client");
 const reducers = require("./reducers");
 const selectors = require("./selectors");
 
-const App = require("./components/App");
+const App = require("./components/App").default;
```

Comment:
I thought I'd do that when I migrate main.js file. But essentially the same thing.